dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7626,6 +7646,19 @@ public String toString() {
       stringBuilder(Name.ZOOKEEPER_JOB_LEADER_PATH)
           .setDefaultValue("/alluxio/job_leader").build();
 
+  //
+  // Membership related properties
+  //
+  public static final PropertyKey ALLUXIO_CLUSTER_NAME =
+      stringBuilder(Name.ALLUXIO_CLUSTER_NAME)
+          .setDefaultValue("DefaultAlluxioCluster").build();
```

Comment:
why aren't all the builder fields being set? specifically
- for ALLUXIO_CLUSTER_NAME: setDescription, setScope, setConsistencyCheckLevel
- for WORKER_MEMBERSHIP_ETCD_ENDPOINTS: setScope, setConsistencyCheckLevel

Reply:
oh i think i missed it, let me add it, is it causing any problems now?